### PR TITLE
L-01: `tryDecrease` return initialized value if delta is initialized

### PR DESCRIPTION
**Pull request by arr00**
## Summary by CodeRabbit

* **Bug Fixes**
  * Corrected edge-case handling for uninitialized values in safe arithmetic operations, affecting the success condition and output value behavior.

* **Tests**
  * Updated test cases to validate corrected edge-case scenarios.

### CODE REVIEW

File: test/utils/FHESafeMath.test.ts [hunk 1]
```diff
@@ -61,10 +61,10 @@ describe('FHESafeMath', function () {
     for (const args of [
       // a - b = c & success
       [undefined, undefined, undefined, true],
-      [undefined, 0, undefined, true],
+      [undefined, 0, 0, true],
       [0, undefined, 0, true],
       [1, 1, 0, true],
-      [undefined, 1, undefined, false],
+      [undefined, 1, 0, false],
```
Comment (james-toussaint) [inline test/utils/FHESafeMath.test.ts]: > Otherwise, `success` will be false and `updated` will be the original value.
